src/main/java/bowling/domain/Frame.java

```diff
@@ -32,6 +37,18 @@ public boolean isSpare() {
         return pitches.isSpare();
     }
 
+    public FrameInfo assembleFrameInfo() {
+        return new FrameInfo(assemblePinDownResults(pitches), score(), isFinalFrame(), isSpare(), isScoreDecided());
```

Comment:
```suggestion
        return new FrameInfo(pitches.assemblePinDownResults(), score(), isFinalFrame(), isSpare(), isScoreDecided());
```
```java
//class Pitches
public List<Integer> assemblePinDownResults() {
    return values.stream()
            .map(Pitch::value)
            .collect(Collectors.toList());
}
```
assemblePinDownResults도 Pitches로 이동시킬 수 있겠네요.